server/integration_tests/test_data/international/query_4/chart_config.json

```diff
@@ -245,35 +274,6 @@
               }
             ],
             "title": "Rate of Population Growth in Eurostat Nuts 3 Places of Turkey"
-          },
```

Comment:
this was part of dc/topic/WHOMaternalMortality which got bumped out of the list of matches